app/templates/index.html

```diff
@@ -227,7 +296,7 @@ <h3>Creators...</h3>
                 const style = window.getComputedStyle(infoModal);
                 eligable = style.opacity != '1';
             } catch (error) {
-                console.log(error);
+                // console.log(error);
```

Comment:
Out of curiosity, why are we commenting this out?

Reply:
Handling this for open modals now too.